avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/serialization/AVRO_1_4/StringableRecord_SpecificSerializer_6174384286732341990.java

```diff
@@ -88,6 +88,20 @@ public void serializeStringableRecord0(StringableRecord data, Encoder encoder)
         serializeStringableSubRecord0(subRecord0, (encoder));
         AnotherSubRecord subRecordWithSubRecord0 = ((AnotherSubRecord) data.get(8));
         serializeAnotherSubRecord0(subRecordWithSubRecord0, (encoder));
+        CharSequence stringUnion0 = ((CharSequence) data.get(9));
+        if (stringUnion0 == null) {
+            (encoder).writeIndex(0);
+            (encoder).writeNull();
+        } else {
+            if (stringUnion0 instanceof CharSequence) {
```

Comment:
Why do we have this `instanceof` check? It should always be true, right? Can we get rid of it?

Reply:
It  is not necessary..
I noticed this inefficiency, but I didn't fix it because of laziness..
Thanks for pushing this optimization.